src/Yoti.Auth/CryptoEngine.cs

```diff
@@ -66,7 +65,7 @@ public static byte[] SignDigest(byte[] digestBytes, AsymmetricCipherKeyPair keyp
 
         public static byte[] GetDerEncodedPublicKey(AsymmetricCipherKeyPair keypair)
         {
-            return SubjectPublicKeyInfoFactory.CreateSubjectPublicKeyInfo(keypair.Public).GetDerEncoded();
+            return Org.BouncyCastle.X509.SubjectPublicKeyInfoFactory.CreateSubjectPublicKeyInfo(keypair.Public).GetDerEncoded();
```

Comment:
Could this also be added as `using Org.BouncyCastle.X509.SubjectPublicKeyInfoFactory;`?

Looks fine as is though

Reply:
You can't I'm afraid - `SubjectPublicKeyInfoFactory` is a type, and you can only import namespaces with `using`s. I would be tempted to reinstate `using Org.BouncyCastle.X509` if we were using it in more than one place, but I will leave it as it is since it's only one.